units/src/amount.rs

```diff
@@ -297,36 +299,36 @@ impl OutOfRangeError {
     }
 
     /// Returns true if the input value was large than the maximum allowed value.
-    pub fn is_above_max(&self) -> bool { self.is_greater_than_max }
+    pub fn is_above_max(&self) -> bool { !self.is_negative }
 
     /// Returns true if the input value was smaller than the minimum allowed value.
-    pub fn is_below_min(&self) -> bool { !self.is_greater_than_max }
+    pub fn is_below_min(&self) -> bool { !self.is_negative }
```

Comment:
In 90c5e6f2b7019025afefc78690d563f422fcc808:

Now `is_above_max` and `is_below_min` always return the same value. Previously these two functions were used to distinguish between overshooting the allowable range (for `SignedAmount`) or undershooting the allowable range (for `Amount`, which is what is meant by `is_negative` I guess).

Is this what you intended?

Reply:
Ah this is just a mistake by me. `is_greater_than_max` should be the inverse of `is_negative` but I've forgotten to remove the `!`. Thanks for niticing.

Reply:
No CI fail for a binary mistake - we don't have enough tests. One day I'd really like to put 3 months or so into QA (before or after my "I'd love to put 3 months into miniscript" ... :)

Reply:
I spent 15 minutes trying to add more unit tests then I realised that parsing strings as bitcoin amounts is painful, particular, and uninteresting ... there are better things to do.